test/WebJobs.Script.Tests/Workers/Rpc/GrpcWorkerChannelTests.cs

```diff
@@ -1546,6 +1546,21 @@ await TestHelpers.Await(() => GetInvocationLogs().Length == logLoop,
             }
         }
 
+        [Fact]
+        public async Task NullOutputBinding_DoesNotThrow()
```

Comment:
Fails without change (crashes before logging success), but succeeds with the new change.

Reply:
Ah my other comment touched on this: since this test will fail via thrown exception without your changes, I don't think you need any of the logs to validate the change. Just a lack of thrown exception (and thus test passes) is good enough.